pkg/daemon/daemon.go

```diff
@@ -261,7 +271,15 @@ func cmdRun(p *ptpProcess) {
 		for scanner.Scan() {
 			output := scanner.Text()
 			fmt.Printf("%s\n", output)
-			extractMetrics(p.name, p.iface, output)
+			out := fmt.Sprintf("%s\n", output)
```

Comment:
nit: configure out before the fmt.Printf and use it there

Reply:
done